src/components/Layout/Layout.tsx

```diff
@@ -1,18 +1,28 @@
+import { ROUTE } from "@constants/route";
+import { RedirectPage } from "@pages/RedirectPage";
 import { Outlet } from "react-router";
 import { Footer } from "../Footer";
 import { Header } from "../Header";
 import { SideBar } from "../SideBar";
 import { Content } from "./Layout.styles";
 
 export function Layout() {
+  const isUserAuthorized = !!localStorage.getItem("user");
+
   return (
     <>
-      <Header />
-      <Content>
-        <SideBar />
-        <Outlet />
-      </Content>
-      <Footer />
+      {!isUserAuthorized ? (
```

Comment:
No need to do convertation twice:

isUserUnauthorized = !localStorage.getItem("user");
{isUserUnauthorized ? (